### ISSUE TITLE
go mod versioning issue

### ISSUE THREAD

**Issue by joeblew99** [foomo/simplecert#3]
````
github.com/foomo/simplecert && go run .
go: github.com/cloudflare/cloudflare-go@v0.9.4 requires
        golang.org/x/lint@v0.0.0-20190511005446-959b441ac422: invalid pseudo-version: does not match version-control timestamp (2019-04-09T20:28:23Z)
````

**Comment by dreadl0ck** [comment]
Please supply more information.

Simplecert is not an executable, it is a package meant for being imported and integrated in your code!
Furthermore, to make use of Go modules you need to set the corresponding environment variable: GO111MODULE=on

Using go run inside the simplecert root should give the following error:

    $ go run .
    go run: cannot run non-main package

**Comment by dreadl0ck** [comment]
Frozen because author is unresponsive.

**Comment by smartinov** [comment]
@dreadl0ck seems as if the go modules are broken

```
➜  simplecert git:(master) go mod verify
go: github.com/cloudflare/cloudflare-go@v0.9.4 requires
	golang.org/x/lint@v0.0.0-20190511005446-959b441ac422: invalid pseudo-version: does not match version-control timestamp (2019-04-09T20:28:23Z)
```

**Comment by dreadl0ck** [comment]
Thanks stefan!

Released a new version 1.4 with a fix.

Verification now works for me:

```
$ go mod verify
all modules verified
```
